scripts/us_bea/states_gdp/states_gdp_industry_statvars.mcf

```diff
@@ -0,0 +1,168 @@
+Node: dcid:USSateQuarterlyIndustryGDP_NAICS_11
```

Comment:
I think this file is generated.

"USSate" -> "USState" here and below.

Reply:
Done.